README.md

```diff
@@ -62,7 +62,19 @@ Visit [RBAC permissions for running the tasks](docs/tasks-rbac-permissions.md) i
 
 ## Examples
 
-#### [Unit Tester Pipeline](examples/pipelines/unit-tester) 
+#### [Windows bios Installer Pipeline](pipelines/windows-bios-installer)
+
+Downloads a Windows ISO file into a PVC and automatically installs Windows by using a custom Answer file into a new base DV.
+
+#### [Windows efi Installer Pipeline](pipelines/windows-efi-installer)
+
+Downloads a Windows ISO file into a PVC and automatically installs Windows 11, or 2k22 with EFI enabled by using a custom Answer file into a new base DV.
```

Comment:
Could be Windows 10 or a prior Windows server version too, no?

Reply:
Other Windows version were not tested with this pipeline

Reply:
But that depends on the answers file, not on the pipeline IMO.

Reply:
Correct, but other OSes are not tested - we can't support it